sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Common/Exceptions/RequestValidationException.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
+{
+    /// <summary>
+    /// Exception class for request validations
+    /// </summary>
+    public class RequestValidationException : Exception
```

Comment:
I'm assuming the new exception type has gone through architect review.   If that is not the case, please ask for a review.

Reply:
Will start a review process

Reply:
Meet with @KrzysztofCwalina, since the exceptions are internal **now**, an architect not required for this item.